src/shell.js

```diff
@@ -49,7 +49,8 @@ var Module = typeof {{{ EXPORT_NAME }}} != 'undefined' ? {{{ EXPORT_NAME }}} : {
 #include "polyfill/objassign.js"
 #endif
 
-#if WASM_BIGINT && MIN_SAFARI_VERSION < 150000
+#if WASM_BIGINT && MIN_SAFARI_VERSION < 140100
+// TODO: Fix this back to 150000
 // See https://caniuse.com/mdn-javascript_builtins_bigint64array
 #include "polyfill/bigint64array.js"
```

Comment:
side note; I wasn't even aware we supported WASM_BIGINT without bigint64array (and I even said so in https://webassembly.org/features/). I don't know how many users we had for this; should we just remove the polyfill and don't support bigint for Safari 14?